samples/boards/nordic/clock_control/configs/cpuapp_hsfll.conf

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-CONFIG_CLOCK_CONTROL_NRF2=y
+CONFIG_CLOCK_CONTROL=y
```

Comment:
~This should no longer be needed since its enabled by default :)~ we can add a prj.conf to the sample enabling clock control for all of them, and use `EXTRA_CONF_FILE="configs/fll16m.conf"` to select the specific configs to avoid the repeated config :)